examples/lighting-app/silabs/src/LightingManager.cpp

```diff
@@ -60,9 +65,50 @@ CHIP_ERROR LightingManager::Init()
     }
 
     bool currentLedState;
+
     // read current on/off value on endpoint one.
     chip::DeviceLayer::PlatformMgr().LockChipStack();
     OnOffServer::Instance().getOnOffValue(1, &currentLedState);
+
+#if (defined(SL_MATTER_RGB_LED_ENABLED) && SL_MATTER_RGB_LED_ENABLED == 1)
+    app::DataModel::Nullable<uint8_t> brightness;
+    uint16_t currentx, currenty, currentctmireds;
+    uint8_t currenthue, currentsaturation;
+    Protocols::InteractionModel::Status status;
+
+    // Read brightness value
+    status = Clusters::LevelControl::Attributes::CurrentLevel::Get(1, brightness);
+    if (status == Protocols::InteractionModel::Status::Success && !brightness.IsNull())
```

Comment:
```suggestion
    if (Clusters::LevelControl::Attributes::CurrentLevel::Get(1, brightness) == Protocols::InteractionModel::Status::Success && !brightness.IsNull())
```
that could apply to all if and you could drop the Protocols::InteractionModel::Status status;

Reply:
Modified as per your comment